docs/integration/infra.md

```diff
@@ -0,0 +1,276 @@
+title: Infrastructure Monitoring Integration
+description: Infrastructure performance monitoring - metrics, events, processes, packages, log collection and parsing for bare-metal servers, virtual machines and containers with support for Docker, containerd, k8s, swarm, nomad, ecs, eks, aks, gke.
+
+The Sematext Infra Integration uses [Sematext Agent](../agents/sematext-agent) to collect metrics, events, processes, packages for bare-metal servers, virtual machines and containers. Container monitoring is supported for Docker, containerd, k8s, Swarm, Nomad, ECS, EKS, AKS, GKE and more.
+
+We at Sematext aim to save you time and effort by giving you a strong starting point for monitoring your infrastrcture. You will **not** have to:
+
+- figure out which metrics to collect and which ones to ignore
+- give metrics meaningful labels
+- hunt for metric descriptions in the docs so that you know what each of them actually shows
+- build charts to group metrics that you really want on the same charts, not N separate charts
+- figure out, for each metric, which aggregation to use (min? max? avg? something else?)
+- build dashboards to combine charts with metrics you typically want to see together
+- set up basic alert rules
+
+We set it up for you, out-of-the-box!
+
+
+## Monitoring with Sematext Agent
+
+Sematext Agent collects various metrics and other info about servers, virtual machines, container hosts and containers and ships that to Sematext Cloud.
+
+First create an [Infra Monitoring App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create). You will be presented with installation instructions for many possible environments.
+
+
+## How it works
+
+A Golang-based [Sematext Agent](../agents/sematext-agent) will be installed on your machine. This Agent sends all infrastructure data to an Infra App whose token is configured during the Agent's installation.
```

Comment:
"your machine" could be my Mac, but that's not what we mean, so we don't want anyone to think that.

will be installed on your machine => will be installed on the host you want to monitor

I think we can lowercase agent here and keep it uppercase only when we refer to "Sematext Agent".

Reply:
Yeah, makes sense.